udata_front/theme/gouvfr/assets/js/pages/PublishingForm/Step2DescribeDataset.vue

```diff
@@ -0,0 +1,456 @@
+              <Well class="fr-mt-1w" v-if="stateHasWarning('spatial_information')" color="orange-terre-battue">
+                {{ getWarningText("spatial") }}
+              </Well>
+            </Accordion>
+          </AccordionGroup>
+      </Sidemenu>
+      <div class="fr-col-12 fr-col-md-7">
+        <Container>
+          <Well
+            color="blue-cumulus"
+            weight="regular"
+            class="fr-mb-2w"
+          >
+            <div class="fr-grid-row">
+              <div class="fr-col-auto fr-mr-3v">
+                <img :src="editIcon" alt="" />
+              </div>
+              <div class="fr-col">
+                <p class="fr-m-0 fr-text--bold">{{ $t('What is a dataset?') }}</p>
+                <p class="fr-m-0 fr-text--xs">{{ $t('On {site}, a dataset is a set of files.', {site: title}) }}</p>
+              </div>
+            </div>
+          </Well>
+          <fieldset class="fr-fieldset" aria-labelledby="description-legend">
+            <legend class="fr-fieldset__legend" id="description-legend">
+              <h2 class="subtitle subtitle--uppercase fr-mb-3v">
+                {{ $t("Description") }}
+              </h2>
+            </legend>
+            <LinkedToAccordion
+              class="fr-fieldset__element"
+              :accordion="nameDatasetAccordionId"
+              @blur="vWarning$.title.$touch"
+            >
+              <InputGroup
+                :aria-describedby="nameDatasetAccordionId"
+                :label="$t('Dataset name')"
+                :required="true"
+                v-model="dataset.title"
+                :hasError="stateHasError('title')"
+                :hasWarning="stateHasWarning('title')"
+                :errorText="getErrorText('title')"
+              />
+            </LinkedToAccordion>
+            <LinkedToAccordion
+              class="fr-fieldset__element"
+              :accordion="addAcronymAccordionId"
+              @blur="vWarning$.acronym.$touch"
+            >
+              <InputGroup
+                :label="$t('Acronym')"
+                v-model="dataset.acronym"
+              />
+            </LinkedToAccordion>
+            <LinkedToAccordion
+              class="fr-fieldset__element"
+              :accordion="writeAGoodDescriptionAccordionId"
+              @blur="vWarning$.description.$touch"
+            >
+              <InputGroup
+                :label="$t('Description')"
+                :required="true"
+                type="textarea"
+                v-model="dataset.description"
+                :hasError="stateHasError('description')"
+                :hasWarning="stateHasWarning('description')"
+                :errorText="getErrorText('description')"
+              />
+            </LinkedToAccordion>
+            <LinkedToAccordion
+              class="fr-fieldset__element"
+              :accordion="useTagsAccordionId"
+              @blur="vWarning$.tags.$touch"
+            >
+              <MultiSelect
```

Comment:
When adding tags (or some other properties such as format for remote resource), we probably should be able to add a tag that is not in the suggested list.

Reply:
What are the properties that should allow this ?

Reply:
hmm I would say:
* tag on dataset
* format on remote resource
* mime type on remote resource